tests/test_icons_products_to_deploy_QCI96.py

```diff
@@ -0,0 +1,17 @@
+# https://polarion.engineering.redhat.com/polarion/redirect/project/QuickstartCloudInstaller/workitem?id=QCI-96
+
+# In the "Select products to deploy" screen, when you hover over the "i" icon by each product, text about the product is displayed. This is true whether you have the product selected or not.
+
+def test_icons_products_to_deploy(new_deployment_pg):
+    rhv_expected_text = ("Complete enterprise virtualization management for"
```

Comment:
Do we really want to hard code what text to look for?   I could see them changing this text for any number of reasons.   I realize that makes the ability to automate the test somewhat weak, but I can see us revisiting this a lot, and unexpectedly.